workflow/usage.rst

```diff
@@ -138,7 +138,7 @@ like this:
 
     class BlogPost
     {
-        // This property is used by the marking store
+        // This property is used by the marking store, by default, Symfony "mark" this attribute as an array.
```

Comment:
I don't understand what this means --> _"Symfony mark this attribute as an array"_  Is this something that can only be understood if you use the Workflow component (which I don't) ... or is it supposed to be understood by all readers? Thanks!

Reply:
May be this is better:
> By default symfony set the marking as an array. 
as or with, I don't know